src/sentry/runner/commands/createuser.py

```diff
@@ -58,95 +51,97 @@ def _set_superadmin(user):
 @click.command()
 @click.option("--email")
 @click.option("--password")
-@click.option("--superuser/--no-superuser", default=None, is_flag=True)
-@click.option("--superadmin/--no-superadmin", default=False, is_flag=True)
-@click.option("--staff/--no-staff", default=None, is_flag=True)
+@click.option("--superuser/--no-superuser", default=None, is_flag=True, help="Superusers have full access to Sentry, across all organizations.")
+@click.option("--staff/--no-staff", default=None, is_flag=True, help="Staff users have access to Django backend.")
 @click.option("--no-password", default=False, is_flag=True)
 @click.option("--no-input", default=False, is_flag=True)
-@click.option("--force-update", default=False, is_flag=True)
+@click.option("--force-update", default=False, is_flag=True, help="If true, will update existing users.")
 @configuration
-def createuser(email, password, superuser, superadmin, staff, no_password, no_input, force_update):
+def createuser(email, password, superuser, staff, no_password, no_input, force_update):
     "Create a new user."
 
-    from django.conf import settings
+    if settings.SENTRY_SELF_HOSTED or settings.SENTRY_SINGLE_ORGANIZATION:
```

Comment:
nit: I like a negative here for an early exit, clear short-circuit and avoids excessive indentation below.

```suggestion
    if not (settings.SENTRY_SELF_HOSTED or settings.SENTRY_SINGLE_ORGANIZATION):
        return
```